core/src/main/java/com/auth0/authentication/AuthenticationAPIClient.java

```diff
@@ -447,7 +447,7 @@ public ChangePasswordRequest changePassword(String email) {
                 .addParameters(parameters);
     }
 
-    protected ParameterizableRequest<Token> loginWithResourceOwner() {
+    public ParameterizableRequest<Token> loginWithResourceOwner() {
```

Comment:
I reverted the change to make them protected just to keep all Lock.Android functionality.
I'll add javadoc then